internal/slack/slack.go

```diff
@@ -368,3 +371,37 @@ func (c *Client) NotifyK8SPodErrorEvent(ctx context.Context, event *http.PodErro
 	}
 	return err
 }
+
+func (c *Client) NotifyReleaseManagerError(ctx context.Context, msgType, service, environment, branch, namespace, actorEmail string, err error) error {
+	if c.muteOptions.ReleaseManagerError {
+		return nil
+	}
+	userID, err := c.getIdByEmail(ctx, actorEmail)
+	if err != nil {
```

Comment:
I think it's totally fine to just return the `err`, as we handle the user email lookup and error logging in `c.getIdByEmail(ctx, actorEmail)`

Reply:
that will instead result in an error here:
https://github.com/lunarway/release-manager/pull/185/files/400949c1666eb8fe2849362e6a5b4274f37d143c#diff-2481863f7833169b61254081a2d3aedcR319-R322

Is that better?

Reply:
That's up to you :)